docs/macos_documentation/installation.md

```diff
@@ -0,0 +1,24 @@
+# MacOS Installation Guide
+
```

Comment:
Can you help add something like:
If the following error appears, you can use the following instructions:

Reply:
I've updated the document and added the related error message.